public/store/dataset/actions.ts

```diff
@@ -265,6 +265,8 @@ export const actions = {
       status: DatasetPendingRequestStatus.PENDING
     };
 
+    const sheblag = getters.getVariables(context);
```

Comment:
This looks like it's unused - I think we can cut it?

Reply:
Yeah, left over debugging cruft.  I'll get it now.